<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiMenu</class>
 <widget class="QDialog" name="WifiMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1845</width>
    <height>1054</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pButton_brock">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>230</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/CancelTest.png);</string>
   </property>
   <property name="text">
    <string>    断开连接</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_refresh">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>420</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/confirmbt.png);</string>
   </property>
   <property name="text">
    <string>    刷新列表</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_return">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>610</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string>    返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 206, 206);</string>
   </property>
   <property name="text">
    <string>可用WLAN列表</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_wifi">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>791</width>
     <height>651</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(211, 0, 0);
selection-color: rgb(204, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>801</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/wifibg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>pButton_brock</zorder>
  <zorder>pButton_refresh</zorder>
  <zorder>pButton_return</zorder>
  <zorder>tableWidget_wifi</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
